xcarchive/ios_test.go

```diff
@@ -24,7 +24,10 @@ func sampleRepoPath(t *testing.T) string {
 		require.NoError(t, err)
 		sampleArtifactsGitURI := "https://github.com/bitrise-samples/sample-artifacts.git"
 		cmd := command.New("git", "clone", sampleArtifactsGitURI, dir)
-		require.NoError(t, cmd.Run())
+		_, err = cmd.RunAndReturnTrimmedCombinedOutput()
+		if err != nil {
+			t.Errorf("git clone failed: %s", err)
+		}
```

Comment:
It is not clear for me why this changes is needed. Could you tell me why?

Reply:
Tests are failing in CI with the CI git account. The old code didn't provide a meaningful error message, I had to add this to see the root cause. But it's still work in progress, please ignore for now :)